nixos/modules/services/web-servers/garage.nix

```diff
@@ -49,7 +49,11 @@ in
 
           data_dir = mkOption {
             default = "/var/lib/garage/data";
-            type = types.path;
+            example = [ {
+              path = "/var/lib/garage/data";
+              capacity = "2T";
+            } ];
+            type = with types; either path (listOf attrs);
```

Comment:
The description doesn't cover the new syntax yet.

Reply:
It didn't cover the old syntax neither. Can you suggest wording?

Reply:
The directory in which Garage will store the data blocks of objects. This folder can be placed on an HDD.
Since v0.9.0, Garage supports multiple data directories, refer to https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#data_dir for the exact format.

Reply:
I honestly think we should remove most of these options alltogether, we're doing a bad job of copying half of the options and their documentation from the manual, causing problems whenever the format change and we don't update it. See also https://github.com/NixOS/nixpkgs/pull/309643.

Though this is probably out of scope for this PR.